hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1193,6 +1218,21 @@ public Builder withMergeAllowDuplicateOnInserts(boolean routeInsertsToNewFiles)
       return this;
     }
 
+    public Builder withFileIOBufferEnabled(boolean enabled) {
+      props.setProperty(FILE_IO_BUFFER_ENABLED, String.valueOf(enabled));
+      return this;
+    }
+
+    public Builder withFileIOBufferMinSize(int sizeInBytes) {
```

Comment:
can we please try to be uniform. for getter methods, we have suffixed "ing" for buffer, but here we haven't. was it a conscious decision?

Reply:
Buffering is a verb. So when asking "is buffering enabled" I have used the ing form. The size of the buffer is in bytes. So for getting the size of the buffer, I have used "buffer". 

Buffering is enabled/disabled. Buffering does not have a size. Buffer has a size.